pkg/build/scanner.go

```diff
@@ -88,6 +89,50 @@ func Scanner(manifest model.Manifest) error {
 		return nil
 	}
 
-	// There were vulnerabilities. Return body.
-	return errors.New("Image name: " + baseImageName + "\n" + string(body))
+	// There were vulnerabilities. Output message listing vulnerabilities.
+	log.Infof("Base image scan of %s failed with:\n %s", baseImageName, string(body))
+
+	// If IgnoreVulernability is true, just just return
+	if manifest.IgnoreVulnerability == true {
+		return nil
+	}
+
+	// Else build a new set of images.
+	// baseVersion is like 1.10-dev.1
+	// Increment the digit after the last period to get the new tag.
+	index := strings.LastIndex(baseVersion, ".") + 1
+	var digit int
+	if digit, err = strconv.Atoi(baseVersion[index:]); err != nil {
+		return err
+	}
+	newBaseVersion := baseVersion[:index] + strconv.Itoa(digit+1)
+	log.Infof("new baseVersion: %s", newBaseVersion)
+
+	// Attempted to run the ./tools/build-base-images.sh but locally, that gave me a
+	// number of different failures related to docker, PATHS, TTY, etc.
+	// Instead run the make command similar to what the script does.
+	buildEnv := []string{
+		"HUBS=docker.io/istio gcr.io/istio-release",
+		"TAG=" + newBaseVersion,
+		"BUILDX_BAKE_EXTRA_OPTIONS=--no-cache --pull",
+		"DOCKER_TARGETS=docker.base docker.distroless docker.app_sidecar_base_debian_9 docker.app_sidecar_base_debian_10 docker.app_sidecar_base_ubuntu_xenial docker.app_sidecar_base_ubuntu_bionic docker.app_sidecar_base_ubuntu_focal docker.app_sidecar_base_centos_7 docker.app_sidecar_base_centos_8",
```

Comment:
this will make it a bit hard to maintain?

Reply:
Agreed and I preferred to use the script. The script just lead to a lot of different failures. I could go back to trying the script and all the various RunMake changes around the `cmd` again.

Reply:
If I get it in the change to verify the images doesn't exist in the script, and can add a number of changes to actually get the script to work here, that should be good.